packages/bpk-component-popover/src/BpkPopoverPortal.js

```diff
@@ -73,35 +62,37 @@ class BpkPopoverPortal extends Component {
       return;
     }
 
-    const options = {
-      classPrefix: 'bpk-popover-tether',
-      element: popoverElement,
-      target: targetElement,
-      ...applyRTLTransforms(this.props.tetherOptions),
-    };
-
-    if (!this.tether) {
-      this.tether = new Tether(options);
-      this.tether.on(
-        'position',
-        getArrowPositionCallback(
-          popoverElement,
-          ARROW_ID,
-          'bpk-popover-tether',
-        ),
-      );
-    } else {
-      this.tether.setOptions(options);
+    if (!this.popper || targetElement !== this.previousTargetElement) {
+      if (this.popper) {
+        this.popper.destroy();
+      }
+
+      this.popper = new Popper(targetElement, popoverElement, {
+        placement: this.props.placement,
+        onCreate: () => {
+          targetElement.focus();
```

Comment:
This logic has been moved from `onOpen` to here because as popper.js's update happens asynchronously which means the focus call causes a jump in the page. Moving it to `onCreate` means we are certain that popper.js has finished.